src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,17 +157,24 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
+    auto output = LedgerEntryHandler::Output{};
+    output.deletedLedgerIndex = std::nullopt;
+    auto ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
+
+    if (!ledgerObject || ledgerObject->empty()) {
+        if (not input.includeDeleted) 
+            return Error{Status{"entryNotFound"}};
+        ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq - 1, ctx.yield);
```

Comment:
lgrInfo.seq is the seq specified by the input, right? It is not the seq for the ledgerObject.

for example if object A got deleted at seq 100 but created at seq 80
in db, objects table is like:
key 100 null
key 80 0x1234
If the seq from input request is 120: 
First, we got the null as ledgerObject
Second, we shall use fetchLedgerObject(key,**99**, ctx.yield); to get 0x1234. If we use lgrInfo.seq - 1 which is 119, we will still get null, right?